shell/config/router/navigation-guards/clusters.js

```diff
@@ -151,9 +111,7 @@ export default async function({
       })
     ]);
 
-    if (localCheckResource) {
```

Comment:
This check doesn't appear to be needed any longer. It seemed to be necessary because